packages/rich-text/src/component/index.native.js

```diff
@@ -586,6 +586,15 @@ export class RichText extends Component {
 	}
 
 	willTrimSpaces( html ) {
+		const {
+			tagName,
+		} = this.props;
+
+		// aztec won't trimm spaces in a case of <pre> block, so we are excluding it
+		if ( tagName === 'pre' ) {
```

Comment:
Nice and clear way of shortcircuiting `willTrimSpaces` in the Preformatted block case!

Can you also add a test in https://github.com/WordPress/gutenberg/blob/master/packages/rich-text/src/component/test/index.native.js to cover that case? Thanks!

Reply:
Good suggestion, thanks!